addons/docs/aurelia/index.js

```diff
@@ -1 +0,0 @@
-module.exports = require('../dist/frameworks/aurelia/index');
```

Comment:
Note: deleted because there is no `aurelia/index` file

Reply:
+ it all moves to `addons/docs/common-preset.js`

Reply:
we probably should've deleted all these files in 6.0 ... i should fix in 7.0 🙈